src/Koffee/MainModel.fs

```diff
@@ -462,6 +463,7 @@ type MainEvents =
         OpenParent, "Open Parent Folder"
         Back, "Back in Location History"
         Forward, "Forward in Location History"
+        StartPrompt ShowHistory, "Show History"
```

Comment:
Please rename and update description to `ShowLocationHistory` to avoid confusion with other kinds of history.